pkg/apis/atlassian/v1/cyclenoderequest_types.go

```diff
@@ -181,8 +181,9 @@ type CycleNodeRequest struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec   CycleNodeRequestSpec   `json:"spec,omitempty"`
-	Status CycleNodeRequestStatus `json:"status,omitempty"`
+	ClusterName string                 `json:"clusterName,omitempty"`
```

Comment:
Was the addition of `ClusterName` intended?

Reply:
Yeah apimachinery removed the field in v0.25.x https://github.com/kubernetes/apimachinery/blob/v0.25.11/pkg/apis/meta/v1/types.go#L258C34-L260
The field was removed since it wasn't needed by kubernetes
>This field is not set anywhere right now and apiserver is going to ignore it if set in create or update request.

While we don't need it for any kubernetes functionality, we still need it since we piggybacked the meta clusterName field for cyclops notification components.

Reply:
Would this place the field inside the `metadata` field though?

Reply:
No it will be just under `clustername` sine we will (un)marshal it the same way where it's used by the notification https://github.com/atlassian-labs/cyclops/blob/master/pkg/notifications/slack/slack.go#L80-L91. If for convention, we can just create a new struct with the ObjectMeta promoted fields and clustername so we can put it under `metadata`